src/jit/lclvars.cpp

```diff
@@ -1727,12 +1752,27 @@ void Compiler::lvaPromoteStructVar(unsigned lclNum, lvaStructPromotionInfo* Stru
             fieldVarDsc->lvIsRegArg = true;
             fieldVarDsc->lvArgReg   = varDsc->lvArgReg;
             fieldVarDsc->setPrefReg(varDsc->lvArgReg, this); // Set the preferred register
+#if FEATURE_MULTIREG_ARGS && defined(FEATURE_SIMD)
+            if (varTypeIsSIMD(fieldVarDsc))
+            {
+                fieldVarDsc->lvOtherArgReg = varDsc->lvOtherArgReg;
```

Comment:
It is used for multireg args.

---
In reply to: [95663367](https://github.com/dotnet/coreclr/pull/8402#discussion_r95663367) [](ancestors = 95663367)